### DISCUSSION TITLE
Between-Subjects Covariates in Mixed Model

### DISCUSSION THREAD

**Discussion by delucalab** [easystats/parameters#648] [Q&A]
Happy New Year! I had a conceptual question about the inclusion of between-subjects covariates in a model where I am interested in within-subjects differences:

As background, I have paired data regarding the presence and absence of lesions in 3 regions from individual patients and I’m interested in the differences between those regions (ie within-subjects differences). I am running a mixed effects logistic regression with a random intercept for subject and extract the estimated marginal means before making pairwise comparisons between regions. However, I am wondering if there is any conceptual reason to include between-subjects covariates (ie age, sex, etc) in my model? In cases where I am making across-subject comparisons, I know that I typically include these types of covariates so that means are “adjusted”? However, in this case, I’m not making any across-subject comparisons but only within-subjects. Therefore, shouldn’t the paired nature of the data control for those between-subjects factors and including them in the model would be unnecessary?

**Comment by bwiernik** [comment]
It's not really necessary to include them once you have the random intercept, but if those factors are good predictors, they can absorb some of the random effects variance, increasing your statistical power and precision.

**Reply by delucalab** [reply]
Thanks! Along these lines, I was trying to use the performance package using test_performance to compare the models with the covariates and without. However, due to missingness in some of the covariates I get the following error:

Error: The models' dependent variables don't have the same data, which is a prerequisite to compare them. Probably the proportion of missing data differs between models.

Is there another way I can systematically choose whether to include the covariates?

**Reply by bwiernik** [reply]
You need to have the same datasets for any comparisons to make sense. First remove missing data and then fit both models with the same dataset. 

That said, I suggest just including the covariates unless you are losing a lot of sample to missing data. Model comparison isn't necessary here.
